collect_app/src/main/java/org/odk/collect/android/injection/config/AppDependencyModule.java

```diff
@@ -327,7 +327,9 @@ public ODKAppSettingsImporter providesODKAppSettingsImporter(Context context, Pr
                 projectsRepository,
                 settingsProvider,
                 Defaults.getUnprotected(),
+                Defaults.getRemovedUnprotected(),
```

Comment:
Do we need this so that we don't end up in a weird state if the settings don't exist? Like when upgrading from a new (after this) version for instance. I can't seem to make any test fail by passing in `new HashMap<>()` instead.

Reply:
Oh, it's redundant you are right. I've implemented it this way first and then improved [ODKAppSettingsMigrator](https://github.com/getodk/collect/pull/5556/files#diff-e0f2d4b940734742dc5079c53e36d7ee20029640cc8120debd85ca03cd0a77d6R164) by adding:

```
 .withValues(false, null)
                        .toPairs(
                                ProtectedProjectKeys.KEY_SAVE_AS_DRAFT, false,
                                ProtectedProjectKeys.KEY_FINALIZE, true
                        ),
```

and didn't remove that. It's fixed now.